app/main.py

```diff
@@ -1,3 +1,62 @@
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> int:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: int) -> int:
+        if isinstance(other, (Distance, int, float)):
+            self.km += other.km if isinstance(other, Distance) else other
+        return self
+
+    def __mul__(self, other: int) -> int:
```

Comment:
The return type hint for `__mul__` should be `Distance` instead of `int` since it returns a new `Distance` object.